src/network/netdev/xfrm.c

```diff
@@ -0,0 +1,81 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <net/if.h>
+
+#include "netdev/xfrm.h"
+
+static int xfrm_config_verify(
+    NetDev *netdev,
+    const char *filename
+) {
+    Xfrm *x;
+
+    assert(netdev);
+    assert(filename);
+
+    x = XFRM(netdev);
+
+    if (!x->dev) {
+        log_warning("Xfrm NetDev without dev set in %s. Defaulting to 'lo'", filename);
+        if(!((x->dev = strdup("lo")))) {
```

Comment:
please don't do two things at once. There's nothing wrong doing that in two steps.